pages/boac/boac_api_student_page.rb

```diff
@@ -102,14 +102,16 @@ def graduation
 
   def majors
     if sis_profile && sis_profile['plans']
-      sis_profile['plans'].map do |p|
+      majors = sis_profile['plans'].map do |p|
         {
           active: p['status'] == 'Active',
           college: p['program'],
           major: p['description'],
           status: p['status']
         }
       end
+      active_majors = majors.select { |m| m[:active] }
+      active_majors + (majors - active_majors)
```

Comment:
@pfarestveit Is the object to sort active majors first? If so, it might be more concise to write something like:

`majors.sort_by { |m| m[:active] ? 0 : 1 }`

Reply:
@paulkerschen - oh yeah, that's much prettier.  I'll change it.